app/views/layouts/_ae_resolve_options.html.haml

```diff
@@ -13,7 +13,7 @@
         = _("System/Process")
 
       = select_tag('instance_name',
-                   options_for_select(resolve[:instance_names].sort_by(&:downcase),
+                   options_for_select(resolve[:instance_names].map{|request| _(request)}.sort_by(&:downcase),
```

Comment:
This particular select tag is populated by class instances from one particular automation class. We cannot just apply gettext here, it's a content taken from the database.

![Screenshot from 2020-10-29 15-20-45](https://user-images.githubusercontent.com/6648365/97586288-614eba80-19fa-11eb-847b-de3a21fad15b.png)

Reply:
Sorry, I'm not really understanding could you elaborate further

Reply:
Values in the database are assumed to be entered by users, even if the system seeds them somehow.  User-provided values are not to be translated.  So, in order to fix translations on system things that we store in the database, we need a mechanism to determine that they are system values.

Reply:
@DavidResende0 I can't view the above linked issue, so it's hard for me to guess what the motivation behind this change is. Nevertheless, the above change tries to translate this dropdown, right?
![Screenshot from 2020-10-30 11-16-28](https://user-images.githubusercontent.com/6648365/97693500-8e09dd00-1aa1-11eb-93ff-1d518f22da75.png)

And my screenshot in previous post shows that the content of that dropdown comes from one particular (pre-seeded) automation class and its instances (i.e. we cannot translate those).

Reply:
@chessbyte The situation is more complicated though, because there are pre-seeded (or system) strings, which we do translate (for example table headers in GTL reports) and then system strings, which we cannot translate, such as:
1. strings which identify something (like the above class instances)
2. strings which can be edited (or copied) into something new (for example service dialogs)
3. english looking strings like `and` `or` in filter expressions

So in general, I'd say that system strings, which are read-only can be translated. Pre-seeded strings which can be modified -- no.

Reply:
Yes part of the issue was to translate that dropdown which I guess we should do then?

Reply:
> Yes part of the issue was to translate that dropdown which I guess we shouldn't do then?

Nope. I'd vote for keeping this part as it is now.

Reply:
To make my previous post more understandable: we don't want to be translating this dropdown. It contains class instance identifiers, which by definition cannot be translated.

Reply:
so just to 100% confirm, remove this change?

Reply:
Yep.